engine/apps/slack/models/slack_message.py

```diff
@@ -80,7 +80,8 @@ def get_alert_group(self):
                 self.alert_group.slack_message = self
                 self.alert_group.save(update_fields=["slack_message"])
                 return self.alert_group
-            return self.alert.group
+            else:
```

Comment:
what does `_alert_group` (a few lines above) refer to? I see an `alert_group` reference defined, but no `_alert_group`